astropy/modeling/powerlaws.py

```diff
@@ -390,7 +390,8 @@ def _parameter_units_for_data_units(self, inputs_unit, outputs_unit):
 
 class ExponentialCutoffPowerLaw1D(Fittable1DModel):
     """
-    One dimensional power law model with an exponential cutoff.
+    One dimensional power law model with an exponential cutoff, also 
```

Comment:
Remove the extra space at the end of the line (this is why `travis` is failing).